### DISCUSSION TITLE
[Feature Request] Changelog Support for Azure DevOps Repos

### DISCUSSION THREAD

**Discussion by bs2205** [renovatebot/renovate#33193] [Suggest an Idea]
**Summary**
Add support for fetching and displaying changelogs from Azure DevOps repositories in Renovate.

**Why**
Teams using Azure DevOps need to see changelogs for dependency updates directly in their workflow. This helps in understanding what changes are coming with each update, reducing the risk of unexpected issues.

**Benefits**
Better visibility into dependency changes.
Easier tracking of updates and potential breaking changes.
Improved team collaboration and communication.

**Comment by rarkins** [comment]
https://github.com/renovatebot/renovate/issues/18324
